core/src/main/scala/neotypes/implicits/mappers/ResultMappers.scala

```diff
@@ -4,15 +4,15 @@ package implicits.mappers
 import java.time._
 import java.util.UUID
 
-import mappers.{ResultMapper, TypeHint, ValueMapper}
-import types.Path
-
+import neotypes.exceptions.MultipleIncoercibleException
+import neotypes.mappers.{ResultMapper, TypeHint, ValueMapper}
+import neotypes.types.Path
 import org.neo4j.driver.internal.types.InternalTypeSystem
 import org.neo4j.driver.internal.value.IntegerValue
 import org.neo4j.driver.v1.Value
-import org.neo4j.driver.v1.types.{Entity, IsoDuration, Node, Path => NPath, Point, Relationship}
-import shapeless.{HList, HNil, LabelledGeneric, Lazy, Witness, labelled, :: => :!:}
+import org.neo4j.driver.v1.types.{Entity, IsoDuration, Node, Point, Relationship, Path => NPath}
 import shapeless.labelled.FieldType
+import shapeless.{HList, HNil, LabelledGeneric, Lazy, Witness, labelled, :: => :!:}
```

Comment:
It seems you have an auto-formatter which mess with the imports.

I believe the best would be to disable it for this PR, if you want, we can discuss in the gitter room if you believe we should be following another standard. And if get to an agreement on that, open a new PR formatting all the files at once.

Reply:
Whoops, yeah I'll disable it.